Decred-Never-Stops.md

```diff
@@ -0,0 +1,100 @@
+
+iii) Bison Relay: In a world where online content is often subject to censorship, surveillance, and gatekeeping, Bison Relay offers a refreshing alternative. This groundbreaking platform introduces a new paradigm for social media, where freedom of expression reigns supreme. By prioritizing privacy, transparency, and user empowerment, Bison Relay creates a space where individuals can freely share their thoughts, ideas, and experiences without fear of suppression or manipulation.
+
+Decred contributors are committed to cultivating a vibrant ecosystem that not only encourages innovation but also embodies values like transparency, autonomy, and user empowerment. Decred's new paradigm of work integrates individuals with valuable skills and ideas from around the world, ensuring fair compensation for those responsible for enhancing the project and introducing new concepts. Learn how to become a Decred contributor here! (https://docs.decred.org/contributing/overview/)
+
+Outreach efforts
+
+Just like any other crypto project Decred does not maintain itself just with development, but also with outreach efforts coordinated by its community members. Those efforts have two main goals, which are: i)spreading the Decred word, making the project and its ethos known by the crypto community and investors; and ii)Explaining Decred, keeping the community and interested people informed about the project ethos, latest updates, and researches that are being made. 
+
+The Vanguard team is a community-driven initiative aimed at boosting Decred's presence across social media platforms. Contributors are motivated to strategize, organize, and amplify Decred's message. Anyone passionate about advancing DCR's growth is welcome to join and contribute to the cause. Tivra stands out as one of the most engaged members of Vanguard. Beyond spreading the word about Decred, he recognizes the immense value in connecting with various crypto communities and appreciates the unique approaches each brings to the table: 
+
+"My outreach efforts via the Vanguard proposal involve making genuine connections within the crypto communities with similar principles. For instance, engaging in Monero Spaces led to meeting Vik Sharma and the incredible opportunity for Cake Wallet integration.  That kind of open collaboration speaks to Decred's spirit as an adaptive DAO – we learn, evolve, and find synergies where others might see only competition.  Decred's strength lies in community governance, and I strongly believe outreach means bringing that model to more people, and welcoming new stakeholders who will shape its future."
+
+A new Vanguard proposal is being voted on right now. Check how is it going here ([link to the new Vanguard proposal](https://proposals.decred.org/record/d658f9a)).
```

Comment:
Once again, I'd probably change this to something like "You can read more about the Vanguard proposal here - link-"